nifi-registry/nifi-registry-docker-maven/dockermaven/pom.xml

```diff
@@ -61,6 +61,22 @@
                                     <goal>run</goal>
                                 </goals>
                             </execution>
+                            <execution>
+                                <id>copy-toolkit-for-docker</id>
+                                <phase>process-sources</phase>
+                                <configuration>
+                                    <target name="copy toolkit assembly to nifi-registry-docker for image build">
```

Comment:
I think it would better to use the approach used in the main docker image where the NiFi binary is downloaded rather than pulled from the nifi-assembly package.

https://github.com/apache/nifi/blob/3d60e4a4f602b88686806ee84d1da816c9c398a2/nifi-docker/dockerhub/Dockerfile#L58

Reply:
This change is to build a registry container image all from the source code, if there is a change in nifi registry code it will also be update the image with that change. By that way you should refer Dockerfile at [nifi-docker/dockermaven/Dockerfile](https://github.com/apache/nifi/blob/3d60e4a4f602b88686806ee84d1da816c9c398a2/nifi-docker/dockermaven/Dockerfile#L43) but not [nifi/nifi-docker/dockerhub/Dockerfile](https://github.com/apache/nifi/blob/3d60e4a4f602b88686806ee84d1da816c9c398a2/nifi-docker/dockerhub/Dockerfile#L58)

Reply:
I also updated the code to include changes to [nifi-registry/nifi-registry-core/nifi-registry-docker/dockerhub/Dockerfile](https://github.com/apache/nifi/blob/main/nifi-registry/nifi-registry-core/nifi-registry-docker/dockerhub/Dockerfile) which is used to publish registry official image to hub.docker.com. Please help to review.